src/app/payload-decoder/payload-decoder-edit/payload-decoder-edit.component.ts

```diff
@@ -60,10 +60,7 @@ export class PayloadDecoderEditComponent implements OnInit {
   public application: Application;
   public iotDevices: IotDevice[];
   public iotDevice: IotDevice;
-  public pageLimit = environment.tablePageSize;
-  public pageTotal: number;
-  public pageOffset = 0;
-  public deviceSubscription: Subscription;
+    public deviceSubscription: Subscription;
```

Comment:
(Very) minor comment, we'll just change it after merging into stage:
Please follow the same indentation style as the rest of the document